infra-templates/windows-healthcheck/0/rancher-compose.yml

```diff
@@ -0,0 +1,11 @@
+.catalog:
+    name: "Healthcheck Service"
+    version: "v0.2.0"
+    description: "Rancher healthcheck service"
```

Comment:
Can you add the minimum Rancher version this works in? I might need to have it edited before, but let's start with `v1.5.0-alpha-1`